plugins/ufm_log_analyzer_plugin/src/loganalyze/log_analyzers/ibdiagnet_log_analyzer.py

```diff
@@ -33,4 +33,5 @@ def full_analysis(self):
         Returns a list of all the graphs created and their title
         """
         self.print_fabric_size()
-        return []
+        fabric_info = self.get_fabric_size()
```

Comment:
Same thing. Use the infrastructure in the base class to run this.

Reply:
fixed